evap/staff/templates/staff_questionnaire_form.html

```diff
@@ -89,19 +89,27 @@ <h5 class="card-title">{% trans 'Questions' %}</h5>
             rowAdded = function(row) {
                 row.find('a.btn-secondary').remove();
                 applyTomSelect(row.find("select").get());
+                registerSelectChangedHandlers();
             };
             makeFormSortable("question_table", "questions", rowChanged, rowAdded, "", true, true);
 
-            // enable or disable checkbox for additional text answers
-            $('.question-type select').on('select2:select', function (e) {
+            const selectChangedHandler = function(e) {
                 const checkbox = $($(this).closest("td.question-type")).find("input");
-                if (e.params.data.id == 0 || e.params.data.id == 5) {  // 0: Text question; 5: Heading
+                if (e.target.value == 0 || e.target.value == 5) {  // 0: Text question; 5: Heading
                     checkbox.prop("checked", false);
                     checkbox.prop("disabled", true);
                 } else {
                     checkbox.prop("disabled", false);
                 }
-            });
+            };
+
+            const registerSelectChangedHandlers = function() {
+                document.querySelectorAll(".question-type select").forEach(selectElement => {
+                    selectElement.removeEventListener('change', selectChangedHandler);
+                    selectElement.addEventListener('change', selectChangedHandler);
```

Comment:
From mdn/removeEventlistener (emphasis mine):

> While addEventListener() will let you add the same listener more than once for the same type _if the options are different_

So the remove should be redundant, right?